docker/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# This Dockerfile attempts to install dependencies, run the tests and build the pocket-core binary
+# The result of this Dockerfile will put the pocket-core executable in the $GOBIN/bin, which in turn
+# is part of the $PATH
+
+# First build step to build the app binary
+FROM golang:1.12-alpine AS builder
+
+# Install dependencies
+RUN apk -v --update --no-cache add \
+	curl \
+	git \
+	python \
+	py-pip \
+	groff \
+	less \
+	mailcap \
+	gcc \
+	libc-dev \
+	bash && \
+	pip install --upgrade --no-cache awscli s3cmd python-magic && \
+	apk -v --purge del py-pip && \
+	rm /var/cache/apk/* || true
+
+# Environment and system dependencies setup
+ENV POCKET_PATH=/go/src/github.com/pokt-network/pocket-core/
+ENV GO111MODULE="on"
+
+# Create node root directory
+RUN mkdir -p ${POCKET_PATH}
+WORKDIR $POCKET_PATH
+
+#Install node dependencies
+RUN git clone "https://github.com/pokt-network/pocket-core.git" ${POCKET_PATH}
```

Comment:
Shouldn't this clone command specify a branch or so on? I think this might be a little bit more complicated.